server/models/tagsArticle.js

```diff
@@ -0,0 +1,22 @@
+
+
+module.exports = (sequelize, DataTypes) => {
+  const tagsArticle = sequelize.define('tagsArticles', {
```

Comment:
`tagsArticles` is a `joining` table. Is should not be a Model. 

Tags belong to many Articles through the `TagsArticles` joining table.